src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    /// <summary>
+    /// TODO: Fill this out
+    /// </summary>
+    public class DelayedDeliverySettings : ExposeSettings
+    {
+        internal DelayedDeliverySettings(SettingsHolder settings) : base(settings) { }
+
+        /// <summary>
+        /// TODO: Fill this out
+        /// </summary>
+        public void DisableTimeoutManager()
```

Comment:
I understand that we discourage fluent api, as it is no longer trendy and sexy. Let's ignore the fact that we still have fluent api in our code for a moment.

Let's make a step back and put a problem statement. Currently to set both options `DisableTimeoutManager` and `AllEndpointsSupportNativeDelays` person need to write 3 lines of configuration code with some repeated parts. I don't find it intuitive nor easy to use. When I used it I had an impression that I can set only 1 of the 2. I am open to other suggestions, but current state I don't particularly like